pyproject.toml

```diff
@@ -51,10 +51,6 @@ dialect = "postgres"
 [tool.sqlfluff.rules.capitalisation.identifiers]
 capitalisation_policy = "consistent"
 
-[tool.pytest.ini_optons]
-addopts = "-v --cov=layer/nrlf/nrlf --cov-report=term-missing --cov-branch"
-
-
```

Comment:
Removed as was being overwritten by the `[tool.pytest.ini_options]` section further down.